src/commands/integration-test-install-app-platform.yaml

```diff
@@ -17,4 +17,4 @@ steps:
         - run:
             name: "architect/integration-test-install-app-platform: Run apptestctl bootstrap"
             command: |
-              apptestctl bootstrap --kubeconfig="$(kind get kubeconfig)"
+              apptestctl bootstrap --kubeconfig-path=/.kube/config
```

Comment:
The `--kubeconfig-path` flag got added in v0.5.0.

When running `go test` we set this env var. `E2E_KUBECONFIG=~/.kube/config`

I think it's better to use the same approach here for consistency.